pkgs/data/themes/adi1090x-plymouth-themes/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nix-prefetch jq
+
+theme_archives=$(curl --silent https://api.github.com/repos/adi1090x/files/git/trees/master\?recursive=1 \
+  | jq '.tree | map(select(.path| test("^plymouth-themes/themes/pack_.*tar.gz$"))| .path)')
+
+version="unstable-$(date +%Y-%m-%d)"
```

Comment:
This is wrong, the date should be the date of the commit, not when the script was rung

Reply:
This has been set in `default.nix` to the upstream commit date.